src/Altinn.Platform/Altinn.Platform.Storage/Storage/Controllers/InstancesController.cs

```diff
@@ -55,6 +55,7 @@ public class InstancesController : ControllerBase
         /// </summary>
         /// <param name="instanceOwnerPartyId">the instance owner party id</param>
         /// <returns>list of instances</returns>        
+        [Authorize(Policy = "InstanceRead")]
```

Comment:
Should we use a constant value instead of a string?
In case of a policy name change or something similar?
Same for all [Authorize(Policy = "...")]

Reply:
yeah